src/common/modals/InstanceManager/Overview.js

```diff
@@ -107,6 +128,15 @@ const Overview = ({ instanceName }) => {
     );
   };
 
+  const updateGameResolution = (h, w) => {
+    dispatch(
+      updateInstanceConfig(instanceName, prev => ({
+        ...prev,
+        resolution: { height: h, width: w }
+      }))
+    );
+  };
+
```

Comment:
please call parameters width and height and use property value shorthand

Reply:
height and width are already declared in the upper scope, that's why I did this